lib/tasks/peer_review.rake

```diff
@@ -0,0 +1,36 @@
+# NOTE: The assignment cannot be generated in the assignments rake task since
```

Comment:
@WChrisK Investigate this error further. This type of comment is too vague to belong.